packages/core/src/util.ts

```diff
@@ -40,6 +40,10 @@ export function objectIsExtensible(obj): boolean {
 }
 
 export function makeBacktrace(stack: string, filterHbSourceCode = false, logger: Logger = console): BacktraceFrame[] {
+  if (typeof stack === 'undefined') {
```

Comment:
Same behavior if `stack` is an empty string? If so, you could just go with `if (!stack)`

Reply:
Yes! Will do that! This is unrelated to this PR but I noticed that I was getting some debug logs that were unnecessary.

Reply:
Done!